src/main/java/com/powsybl/openloadflow/network/impl/AbstractLfBus.java

```diff
@@ -41,7 +37,7 @@
 
     protected boolean voltageControlCapability = false;
```

Comment:
But if the bus is attached to a VoltageControl object, it means is has the capability ? What is the pupose of this property?

Reply:
Property removed